test/integration/editor/index.spec.ts

```diff
@@ -192,3 +192,18 @@ test('code editor auto-complete', async ({ page }) => {
   await page.keyboard.type('textF');
   await expect(page.getByRole('option', { name: 'textField' })).toBeVisible();
 });
+
+test('must deselect selected element when clicking outside of it', async ({ page }) => {
+  const editorModel = new ToolpadEditor(page);
+
+  await editorModel.goToPageById('K7SkzhT');
+
+  await editorModel.waitForOverlay();
+  const nodeHudTag = editorModel.appCanvas.getByTestId('node-hud-tag');
```

Comment:
If would be even better if this node HUD selector could only match the HUD of the `TextField`, but not the HUD of the `Text`.

Reply:
I'm not very good at the `playwright`,
Could you share your idea with me?
thanks

Reply:
thanks.
I got it wrong, I thought I needed to modify the tsx file before